backend/routers/index_router.py

```diff
@@ -0,0 +1,16 @@
+import structlog
+from fastapi import APIRouter, Response
+
+logger = structlog.get_logger()
```

Comment:
You really have to set logger in every router? Why not in `__init__.py`, or just make logger glboal variable

Reply:
I am trying to avoid recursive imports with reexporting from `__init__.py`